app/src/main/java/tech/ula/viewmodel/FilesystemEditViewModel.kt

```diff
@@ -18,28 +29,68 @@ class FilesystemEditViewModel(private val ulaDatabase: UlaDatabase) : ViewModel(
         super.onCleared()
     }
 
+    private val importStatusLiveData = MutableLiveData<FilesystemImportStatus>()
+
+    private var backupUri: Uri? = null
+
+    fun getImportStatusLiveData(): LiveData<FilesystemImportStatus> {
+        return importStatusLiveData
+    }
+
     fun insertFilesystem(filesystem: Filesystem, coroutineScope: CoroutineScope = this) = coroutineScope.launch {
         withContext(Dispatchers.IO) {
             ulaDatabase.filesystemDao().insertFilesystem(filesystem)
         }
     }
 
+    fun getBackupUri(): Uri? {
```

Comment:
I think since we're exposing both a getter and setter for this field that we should just make it public.